setup.py

```diff
@@ -69,22 +76,16 @@ def find_version(file_path):
         'Operating System :: OS Independent',
         'Programming Language :: Python',
         'Programming Language :: Python :: 2',
-        'Programming Language :: Python :: 2.6',
         'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
         # simplejson doesn't support 3.1 or 3.2.
         'Programming Language :: Python :: 3.3',
         'Programming Language :: Python :: 3.4',
+        'Programming Language :: Python :: 3.6',
         'Topic :: Internet :: WWW/HTTP :: Indexing/Search'
     ],
-    install_requires=[
-        'certifi',
-        'elasticsearch>=1.3.0,<2.0.0',
-        'urllib3>=1.8,<2.0',
-        'simplejson>=3.0',
-        'six>=1.4.0,<2.0'
-    ],
-    tests_require=['mock', 'nose>=1.2.1'],
-    test_suite='nose.collector',
+    install_requires=install_requirements('requirements.txt'),
+    tests_require=['mock'] if PY2 else [],
```

Comment:
We can get away with this as long as we don't release a wheel. (Those don't have the opportunity to run this conditional.) So we should be fine for now.